src/test/scala/com/microsoft/hyperspace/index/RefreshIndexTests.scala

```diff
@@ -207,16 +207,45 @@ class RefreshIndexTests extends QueryTest with HyperspaceSuite {
 
       // Replace a source data file with a new file with same name but different properties.
       val deletedFile = deleteOneDataFile(nonPartitionedDataPath)
-      FileUtils.createFile(
-        deletedFile.getFileSystem(new Configuration),
-        deletedFile,
-        "I am some random content :).")
-
-      val ex = intercept[HyperspaceException](
-        hyperspace.refreshIndex(indexConfig.indexName, REFRESH_MODE_INCREMENTAL))
-      assert(
-        ex.getMessage.contains("Index refresh (to handle deleted source data) aborted. " +
-          "Existing source data file info is changed"))
+      val sourcePath = new Path(spark.read.parquet(nonPartitionedDataPath).inputFiles.head)
+      val fs = deletedFile.getFileSystem(new Configuration)
+      fs.copyToLocalFile(sourcePath, deletedFile)
+      val prevIndexLogEntry = getLatestStableLog(indexConfig.indexName)
+      assert(1 == logManager(indexConfig.indexName).getLatestId().get)
```

Comment:
nit: To be consistent with other places: `assert(logManager(indexConfig.indexName).getLatestId().get == 1)`. Same for other places.